public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -51,14 +51,18 @@ export function NoMatchMessage(props: { size: SpacerSize }) {
   );
 }
 
-export function MissingConfigurationMessage(props: {mode: TraceAnalyticsMode}) {
+export function MissingConfigurationMessage(props: { mode: TraceAnalyticsMode }) {
   return (
     <>
       <EuiEmptyPrompt
         title={<h2>Trace Analytics not set up</h2>}
         body={
           <EuiText>
-            {`The indices required for trace analytics (${props.mode === 'jaeger' ? JAEGER_INDEX_NAME : DATA_PREPPER_INDEX_NAME} and ${props.mode === 'jaeger' ? JAEGER_SERVICE_INDEX_NAME : DATA_PREPPER_SERVICE_INDEX_NAME}) do not exist or you do not have permission to access them.`}
+            {`The indices required for trace analytics (${
+              props.mode === 'jaeger' ? JAEGER_INDEX_NAME : DATA_PREPPER_INDEX_NAME
+            } and ${
+              props.mode === 'jaeger' ? JAEGER_SERVICE_INDEX_NAME : DATA_PREPPER_SERVICE_INDEX_NAME
+            }) do not exist or you do not have permission to access them.`}
```

Comment:
Not a huge deal in isolation, but since we're refactoring here, would it clean up code in the area to have this conditional logic moved to better-named variables?

Reply:
Extracted and moved up to make the function itself more clean